src/app/files/fileManager.js

```diff
@@ -136,30 +137,57 @@ class FileManager extends FileSystemApi {
   async setFile (path, content) {
     if (this.currentRequest) {
       let reject = false
-      let savedAsAnotherFile = false
+      let saveAsCopy = false
       let actions = (toaster) => {
-        return yo`<div class="container ml-1">
-        <button class="btn btn-primary btn-sm m-1" onclick=${(e) => { reject = true; e.target.innerHTML = 'Canceled'; toaster.hide() }}>Cancel</button>
-        <button class="btn btn-primary btn-sm m-1" onclick=${(e) => {
-          if (savedAsAnotherFile) return
-          savedAsAnotherFile = true
-          const newPath = path + '.' + this.currentRequest.from
-          this._setFileInternal(newPath, content)
-          this.switchFile(newPath)
-          e.target.innerHTML = 'Saved'
-          toaster.hide()
-        }}>Save As Copy</button>
-        </div>`
+        return yo`
+          <div class="container ml-1">
+            <button class="btn btn-primary btn-sm m-1" onclick=${(e) => {
+              reject = false
+              e.target.innerHTML = 'Accepted'
```

Comment:
I think we should try to adopt a reactive way of programming if possible. In this case you would create a variable `accepted`, and call `yo.update`. Don't change it inside this PR, I'll create an issue for that.